### PR TITLE
feat: 🎸 add vanity url for opensearch cluster

### PR DESCRIPTION
**Pull request by jaskaransarkaria**
tested in ephemeral, see https://et-logs.et.cloud-platform.service.justice.gov.uk/_dashboards

### CODE REVIEW

File: terraform/aws-accounts/cloud-platform-aws/account/opensearch.tf [hunk 1]
```diff
@@ -149,6 +176,16 @@ resource "aws_opensearch_domain" "live_modsec_audit" {
   }
 }
 
+# add vanity url to cluster 
+resource "aws_route53_record" "opensearch_custom_domain" {
+  zone_id = data.aws_route53_zone.et_cloud_platform_justice_gov_uk.zone_id
+  name    = "et-logs"
```
Comment (poornima-krishnasamy) [inline terraform/aws-accounts/cloud-platform-aws/account/opensearch.tf:182]: I think this should be `logs` for the prod account

Comment (jaskaransarkaria) [reply terraform/aws-accounts/cloud-platform-aws/account/opensearch.tf:182]: good catch

File: terraform/aws-accounts/cloud-platform-aws/account/opensearch.tf [hunk 2]
```diff
@@ -81,6 +81,30 @@ resource "aws_kms_key" "live_modsec_audit" {
   multi_region                       = false
 }
 
+data "aws_route53_zone" "et_cloud_platform_justice_gov_uk" {
```
Comment (poornima-krishnasamy) [inline terraform/aws-accounts/cloud-platform-aws/account/opensearch.tf:84]: Just a cosmetic change. Can we name the resource as `cloud_platform_justice_gov_uk`

Comment (jaskaransarkaria) [reply terraform/aws-accounts/cloud-platform-aws/account/opensearch.tf]: good catch

**Review by poornima-krishnasamy — APPROVED** [review]

**Pull request merged**